code/implementation/docker/common/src/main/java/io/cattle/platform/docker/constants/DockerStoragePoolConstants.java

```diff
@@ -0,0 +1,8 @@
+package io.cattle.platform.docker.constants;
+
+public class DockerStoragePoolConstants {
+
+    public static final String DOCKER_KIND = "docker";
+    public static final String DOCKER_FORMAT = "docker";
```

Comment:
could we just default one parameter to another? or is there a chance that docker_format and docker_kind would be different?